### DISCUSSION TITLE
buddhism/2024/seven_full/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#100] [General]
# buddhism/2024/seven_full/

https://shanenull.com/buddhism/2024/seven_full/

**Comment by shane0** [comment]
todo: find or create the chart

a cartesian graph was in this weeks lesson

x axis I'm bad -- I'm good
y axis you are bad you are good

- compassion: i am good you are good, this is the cultivation and unconditioned by circumstances
- arrogant: i am good you are bad
- jealous: you are good i am bad
- depressed: you are bad i am bad

ordinary minds get arrogant jealous depressed cultivated minds stay compassionate unconditioned by circumstances which are marked by impermanence emptiness no self suffering

the self is false the dualistic view of good bad i and you are false views and causes of suffering

everything is changing do all good perform no bad

something I do if i have trouble with a person is find the best memory with them and keep that thought in mind, if there are no good memories or if that does not work then think of something easy like an experience that brought joy with a kind person, and imagine that the cruel person as the kind one, replace the bad person with a good person, until you can do it unconditionally

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2021/favorites/#how-do-we-meditate-on-loving-kindness
